src/main/java/com/google/sps/data/TutorSession.java

```diff
@@ -23,16 +23,21 @@
  */
 public final class TutorSession {
 
+    //an ID counter used for creating IDs for tutor sessions
+    //used for testing only
+    private static long test_id = 0;
```

Comment:
This is a good idea! I struggled to be able to refer to an specific tutoring session in the mock database.

Reply:
Yeah, I noticed we were finding tutor sessions by tutor and student email but then I thought we could probably have more than one session with the same tutor and student emails, so an ID would work better.